src/adr/0032-distributed-mail-queue-cleanup.md

```diff
@@ -0,0 +1,34 @@
+# 32. Distributed Mail Queue Cleanup
+
+Date: 2020-04-13
+
+## Status
+
+Proposed
+
+## Context
+
+Read [Distributed Mail Queue](0031-distributed-mail-queue.md) for full context.
+
+**enqueuedMailsV3** and **deletedMailsV2** is never cleanup and the corresponding blobs are always referenced. This is not
+ideal both from a privacy and space storage costs point of view.
+
+## Decision
```

Comment:
you would end up putting tumbstones in active slices. Sounds harmful. 

I can mention this alternative though. Nice idea ;-)

Reply:
I didn't understand the problem then. Are we talking about calling `delete` on the blobstore? deleting unused slices? or both?

Reply:
The idea would be to do both.

Reply:
Another problem with your proposal: an email is only deleted once, piggy backing deletes on it will not lead to an eventual consistent solution in the face of Cassandra DB failures.